extensions/wikia/IndexingPipeline/PipelineEventProducer.class.php

```diff
@@ -2,25 +2,186 @@
 
 class PipelineEventProducer {
 	const ARTICLE_MESSAGE_PREFIX = 'article';
+	const PRODUCER_NAME = 'MWEventsProducer';
+	const CONTENT = 'content';
```

Comment:
Can we name it more descriptively? Like content namespace or content namespace name?